### PR TITLE
CB-2910 - renaming remaining Ambari things to CM

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

**Comment by attilapalfi92** [comment]
I created a simple datahub cluster and it worked, but haven't tried more complex use cases.

File: core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Stack.java [hunk 1]
```diff
@@ -499,7 +499,7 @@ public void setNetwork(Network network) {
     }
 
     public String getAmbariIp() {
```
[the remark this answers is not in the document]
Comment (attilapalfi92) [reply core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Stack.java]: nope! thx, I fixed it

File: autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java [hunk 2]
```diff
@@ -73,8 +73,8 @@ public void onApplicationEvent(UpdateFailedEvent event) {
             try {
                 String clusterStatus = stackResponse.getCluster().getStatus().name();
                 if (stackStatus.equals(AVAILABLE) && clusterStatus.equals(AVAILABLE)) {
-                    // Ambari server is unreacheable but the stack and cluster statuses are "AVAILABLE"
-                    reportAmbariServerFailure(cluster, stackResponse);
+                    // Clutser manager server is unreacheable but the stack and cluster statuses are "AVAILABLE"
```
[the remark this answers is not in the document]
Comment (attilapalfi92) [reply autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java]: thx, fixed

**Comment by doktoric** [comment]
@keyki can you please double check this pr?

**Pull request merged**